src/tax/hidden-tax-calculation.md

```diff
@@ -12,9 +12,7 @@ _Hidden Tax_ is the amount of VAT that a discount amount has. It will be non-zer
 
 Thus, when there is a discount that has tax embedded in it, Magento computes a _hidden tax_ that is added back in order to compute the discounted price.
 
-```
-discountedItemPrice = fullPriceWithOutTax - discountAmountOnFullPriceWithoutTax + vatAmountOnDiscountedPrice + hiddenTax
-```
+`discountedItemPrice = fullPriceWithOutTax - discountAmountOnFullPriceWithoutTax + vatAmountOnDiscountedPrice + hiddenTax`
```

Comment:
````suggestion
```mathematica
discountedItemPrice = fullPriceWithOutTax - discountAmountOnFullPriceWithoutTax + vatAmountOnDiscountedPrice + hiddenTax
```
````

Reply:
I do prefer the display for the fenced code. I guess since it is sort of an equation we could just call it that language.